Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -79,6 +79,16 @@ def initialize():
     else:
         print "Successfully set owners of certificate folder."
 
+    proxy_paths = ["/etc/opt/microsoft/omsagent/conf/proxy.conf", "/etc/opt/microsoft/omsagent/proxy.conf"]
+    for path in proxy_paths:
+        if os.path.exists(path):
+            process, output, error = linuxutil.popen_communicate(["sudo", "chmod", "g+r", path])
```

Comment:
Ok, thanks for the clarification. It should just be a one-line change to omsadmin.sh, and since it adds permissions, there should be no adverse effects we have to test thoroughly. When you make that change, please reference this PR.